adapter-venice/src/main/resources/venice.md

```diff
@@ -0,0 +1,14 @@
+---
+weight: 0
+title: VeniceDB
+---
+# 1. Overview
```

Comment:
These doc files should contain example ops for each kind of operation a user can configure.

Reply:
I would let @dave2wave fill in this black part as soon as he gets more confidence with this adapter